app/gui/src/dashboard/layouts/Drive/assetsTableItemsHooks.ts

```diff
@@ -76,6 +76,10 @@ export function useAssetsTableItems(options: UseAssetsTableOptions) {
         if (asset.type === AssetType.specialEmpty || asset.type === AssetType.specialLoading) {
           return false
         }
+        // Hide temporary directories of hybrid projects.
+        if (asset.type === AssetType.directory && asset.title.startsWith('cloud-project-')) {
```

Comment:
i think ideally this would be scoped to local backend only. technically also only the default local directory, ideally, but because that's user-defined i don't think that's possible (and i don't think it's desired to bring in more state than preferred)

that said - implementing this might require more info than is passed in so this is fine for now

Reply:
Right, this place is 'too late' to filter out. Ideally we'd rather filter on the pm side.